python/ray/includes/libcoreworker.pxi

```diff
@@ -46,3 +46,48 @@ cdef class ProfileEvent:
         # Deleting the CProfileEvent will add it to a queue to be pushed to
         # the driver.
         self.inner.reset()
+
+
+cdef class ObjectLocation:
```

Comment:
Could we avoid exposing Cython classes in the public API? We can translate this into a pure python class / dict of attributes.